ndn/lpv2/packet.go

```diff
@@ -43,8 +43,10 @@ type Packet struct {
 
 // NewPacket returns an NDNLPv2 frame containing a copy of the provided network-layer packet.
 func NewPacket(fragment []byte) *Packet {
+	fragmentCopy := make([]byte, len(fragment))
+	copy(fragmentCopy, fragment)
```

Comment:
I'm not sure if these copies are necessary. Can we just use the original buffer?

Reply:
No idea, but that's a separate issue. The append was doing the same thing as copy but with a lot of slice grow calls, which showed up in the CPU profile.

Reply:
OK. Then let's forget about it for now.